libs/utils/executor.py

```diff
@@ -519,6 +531,12 @@ def _wload_run(self, exp_idx, experiment):
         self._log.debug('out_dir set to [%s]', experiment.out_dir)
         os.system('mkdir -p ' + experiment.out_dir)
 
+        # Freeze all userspace tasks that we don't need for running tests
+        if self._target_conf_flag(tc, "freeze_userspace"):
```

Comment:
I would prefer (mainly cosmetics) to avid this dummy else branches... can we write this as:
```python
need_thaw = False
if self._target_conf_flag(tc, "freeze_userspace"):
    need_thaw = self._freeze_userspace()
```